### PR TITLE
HALO cloud mask files: engine=netcdf

### PR DESCRIPTION
**Pull request by tmieslinger**
add `engine=netcdf4` to all HALO cloud mask files.
The cloud mask files use `signed bytes` which depend on [this hack](https://github.com/Unidata/netcdf-c/pull/1317), while [opendap does not support signed bytes](https://eurec4a.pages.gwdg.de/how_to_eurec4a/netcdf_datatypes.html#opendap).

### CODE REVIEW

File: HALO/specmacs/cloudmaskSWIR.yaml [hunk 1]
```diff
@@ -122,3 +136,4 @@ sources:
       urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/ESSD_HALO_SPECMACS_CLOUDMASK/EUREC4A_HALO_specMACS-SWIR_cloudmask_20200218_v2.1.nc
       auth: null
       chunks: null
+      engine: netcdf4
```
Comment (d70-t) [inline HALO/specmacs/cloudmaskSWIR.yaml:139]: There's a missing new line at the end of the file.

File: HALO/kt19_cloudmask.yaml [hunk 2]
```diff
@@ -9,87 +9,100 @@ sources:
...
       chunks: null
+      engine: netcdf4
   HALO-0215:
     description: KT-19 cloud mask.
     driver: opendap
     args:
       urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/KT-19/RF_14_20200215/EUREC4A_HALO_KT19_cloudmask_20200215_v4.0.nc
       auth: null
       chunks: null
+      engine: netcdf4
```
Comment (d70-t) [inline HALO/kt19_cloudmask.yaml:108]: there's a missing newline at the end of the file

File: HALO/unified/Radar_cloud_mask_unified.yaml [hunk 3]
```diff
@@ -9,101 +9,116 @@ sources:
...
       chunks: null
+      engine: netcdf4
   HALO-0218:
     description: HAMP Cloud Radar Cloud Mask with unified time coordinate.
     driver: opendap
     args:
       urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/HAMP/cloudmask_v0.9/EUREC4A_HALO_HAMP-Radar_cloud_mask_20200218_v0.9.nc
       auth: null
       chunks: null
+      engine: netcdf4
```
Comment (d70-t) [inline HALO/unified/Radar_cloud_mask_unified.yaml:124]: there's a missing newline at the end of the file

File: HALO/velox_cloudmask.yaml [hunk 4]
```diff
@@ -10,80 +10,92 @@ sources:
...
       chunks: null
+      engine: netcdf4
   HALO-0215:
     description: VELOX cloud mask 
     driver: opendap
     args:
       urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/VELOX/RF_14_20200215/EUREC4A_HALO_VELOX_cloudmask_20200215_v3.0.nc
       auth: null
       chunks: null
+      engine: netcdf4
```
Comment (d70-t) [inline HALO/velox_cloudmask.yaml:101]: there's a missing newline at the end of the file

File: HALO/wales/wales_cloudtop.yaml [hunk 5]
```diff
@@ -9,87 +9,100 @@ sources:
...
       chunks: null
+      engine: netcdf4
   HALO-0215:
     description: WALES-Lidar cloud top height, cloud optical thickness and cloud flag data.
     driver: opendap
     args:
       urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/WALES-LIDAR/2020/2020_02_15/EUREC4A_HALO_WALES_cloudtop_20200215_V1.1.nc
       auth: null
       chunks: null
+      engine: netcdf4
```
Comment (d70-t) [inline HALO/wales/wales_cloudtop.yaml:108]: there's a missing newline at the end of the file

**Review by d70-t — COMMENTED** [review]
Apart from the missing newlines at the end of several files, this looks simple and is a very welcome change.

**Comment by tmieslinger** [comment]
Ups... thanks for the "newline" hint. I added newlines to the end on the mentioned files.

**Review by d70-t — APPROVED** [review]
Thanks for the change. Looks alright now 🎉

**Pull request merged**